SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -406,6 +417,7 @@ private static int CreateSimDeployments(string[] args)
                 {
                     if (simWorkerConfig.worker_flagz[i].worker_type == CoordinatorWorkerName)
                     {
+                        simWorkerConfig.worker_flagz[i].flagz.Add(regionFlag);
```

Comment:
this region flag currently won't be used by the sim player coordinator, right? do we want to keep it around?

Reply:
Smart! I agree!